.env

```diff
@@ -30,4 +30,6 @@ PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 
 IMG_PROXY_URL=https://imgproxy.tools.teritori.com/insecure/
 
+GENERATE_SOURCEMAP=false
```

Comment:
This env suppresses the below warnings:

```
WARNING in ./node_modules/osmojs/node_modules/@cosmjs/proto-signing/build/pubkey.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/pubkey.ts' file: Error: ENOENT: no such file or directory, open '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/pubkey.ts'

WARNING in ./node_modules/osmojs/node_modules/@cosmjs/proto-signing/build/registry.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/registry.ts' file: Error: ENOENT: no such file or directory, open '/Users/sakul/Desktop/repos/teritori-dapp/node_modules/osmojs/node_modules/@cosmjs/proto-signing/src/registry.ts'
```

Reply:
I changed the webapck config to ignore warnings (like in previous expo versions) instead of not generating sourcemaps